pandas/core/groupby/groupby.py

```diff
@@ -2235,8 +2235,11 @@ def _get_cythonized_result(
         grouper = self.grouper
 
         labels, _, ngroups = grouper.group_info
-        output: Dict[base.OutputKey, np.ndarray] = {}
+        output: Dict[base.OutputKey, np.ndarray, str:str] = {}
         base_func = getattr(libgroupby, how)
+        obj = self._selected_obj
+        if isinstance(obj, DataFrame):
+            output["idx_name"] = getattr(getattr(obj, "columns"), "name")
```

Comment:
If we have a `DataFrame`, then it will have a `.columns.name` attribute i.e. don't need the `getattr`

Reply:
Got a bit confused through you mentioning getattr :) Changed it.